File: src/test/java/FileInputReaderTest.java
import org.junit.jupiter.api.Test;
import java.io.IOException;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class FileInputReaderTest {

    @Test
    void testReadInput() throws IOException {
        InputReader fileInputReader = new FileInputReader("src/test/resources/inputFiles/exampleInput.txt");

        String outputString = fileInputReader.readInput();

        assertEquals("Mary had\na little\nlamb", outputString);

    }

}

File: src/test/kotlin/FileInputReaderTest.kt
import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class FileInputReaderTest{
    private val fileInputReader = FileInputReader("./src/test/resources/fileInputReader.txt")

    @Test
    fun `fileInputReader reads text from given resource`(){
        assertNotNull(fileInputReader.read())
        assertTrue(fileInputReader.read().isNotEmpty())
    }
}